theme_cobalt/static/src/scss/primary_variables.scss

```diff
@@ -25,20 +29,28 @@ $o-theme-font-configs: (
 
 $o-website-values-palettes: (
     (
-        'color-palettes-name': 'cobalt-1',
+        'color-palettes-name': 'default-8',
 
         'font': 'Inter',
+        'headings-font': 'Poppins',
 
         'headings-line-height': 1.1,
 
         'btn-ripple': true,
-        'btn-secondary-outline': true,
-        'btn-border-radius': 2px,
+        'btn-border-radius': 10rem,
+        'btn-border-radius-sm': 10rem,
+        'btn-border-radius-lg': 10rem,
+        'btn-padding-y': .5rem,
+        'btn-padding-x': 1rem,
+        'btn-padding-y-lg': 1rem,
+        'btn-padding-x-lg': 2rem,
+        'btn-padding-x-sm': .5rem,
+        'header-template': 'boxed',
+        'header-links-style': 'pills',
+        'footer-template': 'call_to_action',
     ),
 );
 
-$o-selected-color-palettes-names: append($o-selected-color-palettes-names, 'cobalt-1');
```

Comment:
Because `default-8` is already in the palette list so it's not necessary to append it again 👍 
https://github.com/odoo/design-themes/pull/927#discussion_r1772984204